vm/systemSmartContracts/staking.go

```diff
@@ -137,6 +137,14 @@ func (r *stakingSC) Execute(args *vmcommon.ContractCallInput) vmcommon.ReturnCod
 		return r.switchJailedWithWaiting(args)
 	case "getWaitingListIndex":
 		return r.getWaitingListIndex(args)
+	case "getWaitingListSize":
+		return r.getWaitingListSize(args)
+	case "getRewardAddress":
+		return r.getRewardAddress(args)
+	case "getBLSKeyStatus":
+		return r.getStatus(args)
```

Comment:
getBlsKeyStatus

Reply:
I would leave as BLS.

Reply:
I was referred to the name of the method :) to be: getBlsKeyStatus instead getStatus